Contributors.md

```diff
@@ -157,18 +157,7 @@
   </a>
 </td>
 <!-- End of column-13 -->
-<!-- Start of column-14 -->  
```

Comment:
Is this profile on `<!-- Start of column-14 -->` (Olasunkanmi Balogun) a duplicate?